pytest_spec/plugin.py

```diff
@@ -15,6 +15,18 @@ def pytest_addoption(parser):
         help='Print test result in specification format'
     )
 
+    # register config options
+    parser.addini(
+        'spec_header_format',
+        default='{path}:{name}',
```

Comment:
Ok. I renamed it from `{class_name}` since the human readable name I used could also be derived from the module name in case of test *functions*, so I felt like I need something more general.

But [using the raw name](https://github.com/pchomik/pytest-spec/pull/13/commits/4504f51724fee5800481a9344f1d73b20b44f607) instead fixed that.